src/internal/m365/collection/drive/collections_test.go

```diff
@@ -243,7 +243,19 @@ func toPath(elems ...string) string {
 	}
 }
 
-func fullPath(driveID any, elems ...string) string {
+func fullPath(elems ...string) string {
```

Comment:
Totally agree about the design woes here.  I took a "shortest path to victory" approach during the implementation churn (such as in this PR) and intend to rewrite the drive stub handling altogether.  The aim will be to avoid the possibility of gotchas and have a system of factories whose basic design enforces correctness.  Small example: `driveStub.FullPath(...)` instead of `fullPath(arbitraryDriveID, ...)`.  

> switch all...

Heh, jinx: I was already implementing that change while you wrote this review.

> rename...

yup. Will look for ways to not need to know the drive ID outside of handling the drive stub itself.